janusgraph-core/src/main/java/org/janusgraph/diskstorage/keycolumnvalue/scan/StandardScannerExecutor.java

```diff
@@ -98,8 +99,6 @@ public void run() {
         final List<SliceQuery> queries;
         final int numQueries;
         try {
-            job.workerIterationStart(jobConfiguration, graphConfiguration, metrics);
```

Comment:
The changes in this class surely have nothing to do with upgrading to TP 3.3.1 right?

Reply:
@dpitera  3.2.7/3.3.1 added some OLAP profile() fixes which break some tests. Still working through them actually.

Reply:
Does that mean TP tests are not currently all passing with these updates?

Reply:
@sjudeng That is correct.